test/tools/ossfuzz/abiV2FuzzerCommon.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <libsolidity/interface/CompilerStack.h>
+#include <libyul/AssemblyStack.h>
+#include <liblangutil/Exceptions.h>
+#include <liblangutil/SourceReferenceFormatter.h>
+#include <libdevcore/Keccak256.h>
+
+namespace dev
+{
+namespace test
+{
+namespace abiv2fuzzer
+{
+class SolidityExecutionFramework
+{
+public:
+	SolidityExecutionFramework();
+	SolidityExecutionFramework(langutil::EVMVersion _evmVersion);
+
+	Json::Value getMethodIdentifiers()
+	{
+		return m_compiler.methodIdentifiers(m_compiler.lastContractName());
+	}
+
+	dev::bytes compileContract(
+			std::string const& _sourceCode,
+			std::string const& _contractName = "",
```

Comment:
Can be removed, you actually do not even know the name of the contract, do you?

Reply:
You are right, I don't know the name, I can simply refer to it as `m_compiler.lastContractName()`